charts/launch-agent/templates/volcano.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: dependency-install-job
+  annotations:
+    "helm.sh/hook": pre-install
+    "helm.sh/hook-weight": "5"
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  template:
+    spec:
+      containers:
+        - name: volcano-install-job
+          image: bitnami/kubectl:latest
+          command:
+            - "/bin/sh"
+            - "-c"
+            - |
+              kubectl apply -f https://raw.githubusercontent.com/volcano-sh/volcano/master/installer/volcano-development.yaml
```

Comment:
Curious what volcano is?

Reply:
It's a batch scheduler that is becoming widely used for distributed trainings on kubernetes https://volcano.sh/en/

Reply:
We are relying on this to support launching multi node distributed training in k8s

Reply:
Unfortunately they do not have a living helm repo.